config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -33,7 +33,7 @@ presubmits:
         - --ginkgo-parallel=30
         - --provider=gce
         - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gci-gce-ipvs
-        - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
+        - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[sig-storage\] --minStartupPods=8
```

Comment:
We should keep this if we don't want to run storage tests cause `ginkgo.skip`?

Reply:
nvm ignore me, haven't had my morning coffee yet :)

Reply:
this is skipped only for presubmit job, the periodic job will keep running the storage tests